exercises/concept/boutique-inventory/boutique_inventory_test.rb

```diff
@@ -6,19 +6,26 @@ def test_no_item_names
     assert_empty BoutiqueInventory.new([]).item_names
   end
 
-  def test_one_item_name
+  def test_one_item_name_out_of_stock
     items = [
       { price: 65.00, name: "Red Brown Dress", quantity_by_size: {} }
     ]
+    assert_empty BoutiqueInventory.new(items).item_names
+  end
+
+  def test_one_item_name_in_stock
+    items = [
+      { price: 65.00, name: "Red Brown Dress", quantity_by_size: { s: 1 } }
+    ]
     names = ["Red Brown Dress"]
     assert_equal names, BoutiqueInventory.new(items).item_names
   end
 
   def test_three_item_names
     items = [
-      { price: 65.00, name: "Red Brown Dress", quantity_by_size: {} },
-      { price: 50.00, name: "Red Short Skirt", quantity_by_size: {} },
-      { price: 29.99, name: "Black Short Skirt", quantity_by_size: {} }
+      { price: 65.00, name: "Red Brown Dress", quantity_by_size: { s: 1 } },
+      { price: 50.00, name: "Red Short Skirt", quantity_by_size: { m: 1 } },
+      { price: 29.99, name: "Black Short Skirt", quantity_by_size: { l: 1 } }
```

Comment:
Let's not give all items a stock level of 1, but at least retain one with no stock.  It should communicate the idea that it is important to return the item name, regardless of stock level.